### ISSUE TITLE
Trigonal Projectors Error

### ISSUE THREAD

**Issue by SamCox822** [whitead/symd#25]
(Referencing symd/python/symd/groups.py)

In the projectors, I think this line:
`projectors3d["Trigonal"] = projectors3d["Hexagonal"]`
is wrong. 

Trigonal should be **a1 = a2 = a3**, whereas hexagonal is **a1 = a2 != a3**

I think it should be
`projectors3d["Trigonal"] = projectors3d["Cubic"]`

**Comment by whitead** [comment]
I believe this was intentional, but I'm not sure why any more (maybe should have left a comment). Are you sure about your idea? I thought there was some problem like that a cubic box isn't compatible with the way the general positions or the asymmetric unit is defined. Is there some way we can check this.

**Comment by SamCox822** [comment]
I will look into this more. Either way, I am very skeptical that we trigonal is set to hexagonal.

**Comment by whitead** [comment]
You figure this out @SamCox822?

**Comment by SamCox822** [comment]
Here are my thoughts. I saved the projectors (hexagonal, cubic) and applied them like so (to a random numpy array)

```
Hexagonal= np.array(
        [
            3 * [1] + 6 * [0],  # ax
            3 * [-1 / 2] + 6 * [0],  # bx
            9 * [0],  # cx
            9 * [0],  # ay
            3 * [0] + 3 * [np.sqrt(3) / 2] + 3 * [0],  # by
            9 * [0],  # cy
            9 * [0],  # az
            9 * [0],  # bz,
            6 * [0] + 3 * [1],  # cz
        ])

Cubic= np.array(
        [
            3 * [1] + 6 * [0],  # ax
            9 * [0],  # bx
            9 * [0],  # cx
            9 * [0],  # ay
            3 * [1] + 6 * [0],  # by
            9 * [0],  # cy
            9 * [0],  # az
            9 * [0],  # bz,
            3 * [1] + 6 * [0],  # cz
        ])

Cubic = Cubic.reshape(3,3,3,3)
Hexagonal = Hexagonal.reshape(3,3,3,3)
rand = np.random.randint(low=1, high=10, size=(3,3))

rand_hex = Hexagonal@rand
rand_cub = Cubic@rand
```

and got the following outputs:
for Hexagonal@rand:
```
[[[[ 9.         19.         17.        ]
   [ 0.          0.          0.        ]
   [ 0.          0.          0.        ]]

  [[-4.5        -9.5        -8.5       ]
   [ 0.          0.          0.        ]
   [ 0.          0.          0.        ]]

  [[ 0.          0.          0.        ]
   [ 0.          0.          0.        ]
   [ 0.          0.          0.        ]]]


 [[[ 0.          0.          0.        ]
   [ 0.          0.          0.        ]
   [ 0.          0.          0.        ]]

  [[ 0.          0.          0.        ]
   [ 7.79422863 16.45448267 14.72243186]
   [ 0.          0.          0.        ]]

  [[ 0.          0.          0.        ]
   [ 0.          0.          0.        ]
   [ 0.          0.          0.        ]]]


 [[[ 0.          0.          0.        ]
   [ 0.          0.          0.        ]
   [ 0.          0.          0.        ]]

  [[ 0.          0.          0.        ]
   [ 0.          0.          0.        ]
   [ 0.          0.          0.        ]]

  [[ 0.          0.          0.        ]
   [ 0.          0.          0.        ]
   [ 9.         19.         17.        ]]]]
```

```
[[[[ 9 19 17]
   [ 0  0  0]
   [ 0  0  0]]

  [[ 0  0  0]
   [ 0  0  0]
   [ 0  0  0]]

  [[ 0  0  0]
   [ 0  0  0]
   [ 0  0  0]]]


 [[[ 0  0  0]
   [ 0  0  0]
   [ 0  0  0]]

  [[ 9 19 17]
   [ 0  0  0]
   [ 0  0  0]]

  [[ 0  0  0]
   [ 0  0  0]
   [ 0  0  0]]]


 [[[ 0  0  0]
   [ 0  0  0]
   [ 0  0  0]]

  [[ 0  0  0]
   [ 0  0  0]
   [ 0  0  0]]

  [[ 9 19 17]
   [ 0  0  0]
   [ 0  0  0]]]]
```

It seems to me like the last one (which would be cubic) satisfies the a=b=c requirement of trigonal, but maybe not the alpha=beta=gamma!=90 requirement? either way, I think hexagonal is not right for trigonal. 

What do you think?

**Comment by whitead** [comment]
Fixed by 6a0b69eff6f46def67332ee8995337cc0fbb297b
